lib/babe/babe.go

```diff
@@ -185,13 +191,14 @@ func (b *Service) setEpochData(cfg *ServiceConfig, genCfg *types.BabeConfigurati
 		}
 	}
 
-	if cfg.Threshold == nil {
+	if cfg.ThresholdNumerator == nil {
```

Comment:
check for both numerator and denominator being set here, if so then use numerator as C1 and denominator as C2 as input to `CalculateThreshold`